03/app.js

```diff
@@ -1,3 +1,15 @@
 const x = 10;
 let iteration = 0;
-let randomNumber = -1;
\ No newline at end of file
+let randomNumber = -1;
+
+z = Math.round(Math.random() * 20);
```

Comment:
Tutaj nie trzeba było wykorzystywać nowej zmiennej tylko użyć `randomNumber` w warunku.

PS. Jak tworzysz nowe zmienne to pamiętaj o let/const. :)